tests/jenkins/groupvars.yml

```diff
@@ -89,7 +89,6 @@ dataverse:
   postgres:
     reporpm: https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
     version: 9.6
-    jdbcurl: https://jdbc.postgresql.org/download/postgresql-42.2.12.jar
```

Comment:
Pinging @donsizemore to see if he has an opinion on this jdbcurl line being removed.

Reply:
@pdurbin fine by me.